hal/src/msom/hal_platform_config.h

```diff
@@ -31,12 +31,12 @@
 #define HAL_PLATFORM_FLASH_MX25R6435FZNIL0 (1)
 
 #if defined(MODULE_FUNCTION) && MODULE_FUNCTION != 2 // MOD_FUNC_BOOTLOADER
-#define HAL_PLATFORM_USB_PRODUCT_STRING "Mseries"
+#define HAL_PLATFORM_USB_PRODUCT_STRING "M SoM"
 #else
-#define HAL_PLATFORM_USB_PRODUCT_STRING "Mseries DFU Mode"
+#define HAL_PLATFORM_USB_PRODUCT_STRING "M SoM DFU Mode"
 #endif // defined(MODULE_FUNCTION) && MODULE_FUNCTION != 2 // MOD_FUNC_BOOTLOADER
 
-#define PRODUCT_SERIES "Mseries"
+#define PRODUCT_SERIES "M SoM"
```

Comment:
This/these were specifically chosen with the "Series" text.  Are we breaking convention?

Reply:
Changing from Series to some other convention is OK.

My main concern is the space. This constant is used for the BLE advertising name `BleGap::setDeviceName` and the Wi-Fi setup SSID `fetch_or_generate_setup_ssid` (is that still a thing in modern devices?). I'm not sure what issues we introduce with BLE and Wi-Fi stacks if we add a space in the name.

I think it would be safer to define PRODUCT_SERIES as MSoM.